### PR TITLE
add missing dependency

### PR DESCRIPTION
**Pull request by riuvshin**
@vparfonov

### CODE REVIEW

**Comment by vparfonov** [comment]
ok

File: plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/pom.xml [hunk 1]
```diff
@@ -23,6 +23,11 @@
     <name>Che Plugin :: Java :: Plain :: IDE</name>
     <dependencies>
         <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>18.0</version>
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/pom.xml]: version shouldn't be added there (it's part of dependency management)

Comment (vparfonov) [inline plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/pom.xml]: remove version

**Comment by vparfonov** [comment]
ok

**Pull request merged**